### DISCUSSION TITLE
The deprecated api has an option to get milliseconds in the execution date. How does that work with current airflow? With the cli for use with AWS?

### DISCUSSION THREAD

**Discussion by WesternStar** [apache/airflow#18388] [Q&A]
I'm trying to scale airflow and wondering how we increase past the 1 sec per dagrun execution date issue on airflow 2.0. Is this already working with the API? I'm wondering how this would work on AWS in MWAA as well.

**Comment by potiuk** [comment]
I think Airflow 2.2 will bring the ultimate solution - with dag run being decoupled from execution date.  

But I believe even in 2.1 you can specify dag_run _id as you see fit and it is part if the unique key on dag_run via the new REST API of Airflow 2. Not sure if it worked in 2.1.0, but I am quite positive it already worked in 2.1.2

https://airflow.apache.org/docs/apache-airflow/2.1.3/stable-rest-api-ref.html#operation/post_dag_run

I am not sure how it works in MWAA, but you can for sure ask in either MWAA support if you can't find the docs, or in the AWS slack channel in Airflow Slack.
